codes/c/chapter_greedy/max_product_cutting.c

```diff
@@ -0,0 +1,35 @@
+// File: max_product_cutting.c
+// Created Time: 2023-09-15
+// Author: xianii (xianyi.xia@outlook.com)
+
+#include "../utils/common.h"
+
+/* 最大切分乘积：贪心 */
+int maxProductCutting(int n) {
+	// 当 n <= 3 时，必须切分出一个 1
+	if(n <= 3) {
```

Comment:
`if `
The following if also needs to be changed, you can use the formatting tool in vscode.